tdrs-backend/tdpservice/data_files/views.py

```diff
@@ -66,6 +66,10 @@ def create(self, request, *args, **kwargs):
             data_file_id = response.data.get('id')
             data_file = DataFile.objects.get(id=data_file_id)
 
+            logger.debug(f"Preparing parse task: User META -> user: {request.user}, stt: {data_file.stt}. " +
```

Comment:
Is the intention to include most information here? I would only add request.user and data_file_id. For debugging, we still need to dig deeper, however we end up doing the digging

Reply:
Additionally, I think it makes more sense to change this to info